charts/templates/prometheus/serviceMonitor.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.prometheus.enabled }}
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    control-plane: {{ .Values.controlPlane }}
+    {{- with .Values.customLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- include "operator.annotations" . | default "{}" | nindent 4 }}
+  name: controller-manager-metrics-monitor
+  namespace: system
```

Comment:
quick note: you missed this one.
This should either be `namespace: {{- include "operator.namespace" . }}` or configurable